### ISSUE TITLE
Canu correction exit: File size limit exceeded at gatekeeperCreate

### ISSUE THREAD

**Issue by XuewenWangUGA** [marbl/canu#493]
```

- Script:

canu \
 -p $specprefix -d $assem_dir \
 genomeSize=$gEstimateSize \
 -pacbio-raw $indir/$seqf \
 -correct \
 merylMemory=$submem \
 merylThreads=$mythreadsN \
 ovsMemory=$submem \
 useGrid=false


- error information:

-- BEGIN CORRECTION
--
----------------------------------------
-- Starting command on Wed May 10 15:45:33 2017 with 50442.953 GB free disk space

    cd correction
    /usr/local/apps/canu/1.5/bin/gatekeeperCreate \
      -minlength 1000 \
      -o ./pros.gkpStore.BUILDING \
      ./pros.gkpStore.gkp \
    > ./pros.gkpStore.BUILDING.err 2>&1
sh: line 4: 687823 File size limit exceeded/usr/local/apps/canu/1.5/bin/gatekeeperCreate -minlength 1000 -o ./pros.gkpStore.BUILDING ./pros.gkpStore.gkp > ./pros.gkpStore.BUILDING.err 2>&1

-- Finished on Thu May 11 18:32:19 2017 (96406 seconds) with 49171.73 GB free disk space
----------------------------------------
ERROR:
ERROR:  Failed with exit code 153.  (rc=39168)
ERROR:
================================================================================
Don't panic, but a mostly harmless error occurred and Canu stopped.

Canu snapshot v1.5 failed with:
  gatekeeper store exists, but contains no reads
```

**Comment by skoren** [comment]
What does ulimit -a report? Depending on your genome size and coverage, the gatekeeper blobs file (containing sequence) can be 10s of GB so your system has to allow files of that size or you cannot run Canu.

**Comment by XuewenWangUGA** [comment]
Skoren, could you make it more detailed? I am little confused.

**Comment by skoren** [comment]
Run
```
ulimit -a```

and post the output

**Comment by XuewenWangUGA** [comment]
I solved the issue by myself. Now it works.
The issue was caused by many input fasta (my case) files in different locations which were passed by a value containing information of paths and file names to Canu as raw PacBio reads.
My solution is to merge all the fasta or fastq files into one sequence file and then pass the single file to Canu. It works.  
What I did is to "cat" the sequences of the same input files  to a merge sequence file. Then I set the single file in the input parameter of Canu.
